### ISSUE TITLE
Unable to add campaign to uploaded PCAPs

### ISSUE THREAD

**Issue by DaniCea** [crits/crits#732]
`import requests
import os

def _get_file_data(filepath):
    file_obj = open(filepath, 'rb')
    filename = os.path.basename(filepath)
    return {'filedata': (filename, file_obj)}

url = 'URL'
data = {
	'api_key': 'APIKEY',
	'username': 'USERNAME',
	'source': 'Source1',
	'description': 'Description OK',
	'campaign': 'Test',
	'ticket': '45678',
	'bucket_list': 'test,delete'
}
files = _get_file_data('files/pcap1.pcap')
r = requests.post(url, data=data, files=files, verify=False)

print r.__dict__

if r.status_code == 201:
	print "Successfully added "+params['domain']`

Trying to execute the following code, the PCAP is created correctly and it's attached file uploaded succesfully, but the campaign field is completely ignored.

**Comment by apolkosnik** [comment]
Campaign named 'Test' has to be there before you upload.

**Comment by DaniCea** [comment]
Yes, it is there. It works with other elements (Indicator, Exploit...). Anyway, I noticed it doesn't work withg events either, and then I found this issue: https://github.com/crits/crits/issues/616, which I think is probably related
